server/application-server/src/main/java/de/tum/in/www1/hephaestus/gitprovider/pullrequestreview/PullRequestReviewInfoDTO.java

```diff
@@ -18,22 +19,24 @@ public record PullRequestReviewInfoDTO(
         UserInfoDTO author,
         PullRequestBaseInfoDTO pullRequest,
         @NonNull String htmlUrl,
+        @NonNull int score,
         OffsetDateTime submittedAt) {
     // We do not have createdAt and updatedAt for reviews
 
     public static PullRequestReviewInfoDTO fromPullRequestReview(PullRequestReview pullRequestReview) {
         return new PullRequestReviewInfoDTO(
-                pullRequestReview.getId(),
-                pullRequestReview.isDismissed(),
-                pullRequestReview.getState(),
-                pullRequestReview.getComments().size(),
-                UserInfoDTO.fromUser(pullRequestReview.getAuthor()),
-                PullRequestBaseInfoDTO.fromPullRequest(pullRequestReview.getPullRequest()),
-                pullRequestReview.getHtmlUrl(),
-                pullRequestReview.getSubmittedAt());
+            pullRequestReview.getId(),
+            pullRequestReview.isDismissed(),
+            pullRequestReview.getState(),
+            pullRequestReview.getComments().size(),
+            UserInfoDTO.fromUser(pullRequestReview.getAuthor()),
+            PullRequestBaseInfoDTO.fromPullRequest(pullRequestReview.getPullRequest()),
+            pullRequestReview.getHtmlUrl(),
+            (int) ScoringService.calculateReviewScore(List.of(pullRequestReview)),
```

Comment:
Would it be possible to also directly return the int instead of casting it?

Reply:
In the `LeaderboardService` we sum up the scores, so returning `double` makes the sum much more accurate